ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSchedule.java

```diff
@@ -28,10 +28,8 @@
 
   public static final BigInteger DEFAULT_CHAIN_ID = BigInteger.ONE;
 
-  public static ProtocolSchedule create() {
-    return fromConfig(
-        GenesisConfigFile.mainnet().getConfigOptions(), PrivacyParameters.DEFAULT, false);
-  }
+  public static final ProtocolSchedule DEFAULT =
```

Comment:
All of the uses are in test classes.  Can this be moved to one of our test util classes?  Along with `getMainnetConfigOptions`?  Either in an existing or new class in `ethereum/core/src/test-support/java/org/hyperledger/besu/ethereum/core`